src/modules/mount/mount.schema.yaml

```diff
@@ -29,3 +29,12 @@ properties:
                 mountPoint: { type: string }
                 options: { type: string }
             required: [ device, mountPoint ]
+    btrfsSubvolumes:
+        type: array
+        items:
+            type: object
+            additionalProperties: false
+            properties:
+                mountPoint: { type: string }
+                subvolume: { type: string }
+            required: [ subvolume, mountPoint ]
```

Comment:
Thanks for immediately updating the schema. This looks fine.

Reply:
No problem, I was not sure if it was needed so I added it just in case.